MapboxCoreNavigation.podspec

```diff
@@ -40,7 +40,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxNavigationNative", "~> 5.0.0"
+  s.dependency "MapboxNavigationNative", "~> 6.1.1"
```

Comment:
I was going to recommend that MapboxNavigation-Documentation.podspec also be updated, but it doesn’t depend on this pod at all. I wonder how that even works, or whether that’s causing a bug somewhere.